leaflet-markercluster/build.boot

```diff
@@ -0,0 +1,38 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.10.0"  :scope "test"]
+                  [cljsjs/leaflet "1.2.0-0"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "1.3.0")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom  {:project     'cljsjs/leaflet-markercluster
+       :version     +version+
+       :description (str "Provides Beautiful Animated Marker Clustering functionality for Leaflet, a JS library for interactive maps.")
+       :url         "https://github.com/Leaflet/Leaflet.markercluster"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"MIT" "https://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+   (download :url (format "https://unpkg.com/leaflet.markercluster@%s/dist/leaflet.markercluster-src.js.map" +lib-version+)
```

Comment:
Downloading source maps is currently unnecessary because deps-cljs doesn't look for them, and ClojureScript compiler is not able to read source maps for foreign libs.

Reply:
Ok, I've removed the source maps now.